src/sentry/runner/commands/configoptions.py

```diff
@@ -195,19 +195,23 @@ def patch(ctx) -> None:
     if invalid_options:
         status = "update_failed"
         amount = 2
+        ret_val = 2
```

Comment:
I think ret_val needs to be defined outside of the if/else statements otherwise its not in scope on line 214 right?

Reply:
jk I just googled it